Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM alpine:3.9
```

Comment:
NIT: could it be more accurate with the SHA reference? Just wondering if the `alpine:3.9` image might be no deterministic

Reply:
Good idea! I have pinned this to ths SHA of Alpine 3.9. Thank you!